src/port.ts

```diff
@@ -5,9 +5,11 @@ export const port = {
   value: -1,
 }
 
+const DEFAULT_PORT = 1636
```

Comment:
Maybe it's not the best idea to use this port. If the Bee team decides that for some reason they need a new port that would be a natural one to choose for them as well.

I would try to use something that is completely different, like `3636`

Reply:
Or `3054` which is 0xBEE

Reply:
Haha I am sold on the `3054` :-D